src/lib/ledger_catchup/super_catchup.ml

```diff
@@ -952,7 +952,28 @@ let run ~logger ~trust_system ~verifier ~network ~frontier
              [%log debug]
                ~metadata:
                  [("target_hash", State_hash.to_yojson target_parent_hash)]
-               "Failed to download state hashes for $target_hash"
+               "Failed to download state hashes for $target_hash" ;
```

Comment:
Could you special case the no-ancestor error to update the metric since this could be any error? I think we can identify this case in the `try_to_connect_hash_chain` function where we try to connect it to the root hash. Same change in the normal catchup as well